esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2491,7 +2491,28 @@ bool CFileSprayEx::onDespray(IEspContext &context, IEspDespray &req, IEspDespray
             if (isEmptyString(destPlane))
                 destPlane = req.getDestGroup();  // allow eclwatch to continue providing storage plane as 'destgroup' field
             if (isEmptyString(destPlane))
-                throw makeStringException(ECLWATCH_INVALID_INPUT, "Destination storage plane not specified.");
+            {
+                if (destip.isEmpty())
+                    throw MakeStringException(ECLWATCH_INVALID_INPUT, "Neither destination storage plane or destination IP specified.");
+                SocketEndpoint destEp(destip);
+                Owned<IPropertyTreeIterator> planesIter = getDropZonePlanesIterator();
+                ForEach(*planesIter)
+                {
+                    IPropertyTree &lzPlane = planesIter->query();
+                    Owned<IPropertyTreeIterator> hostsIter = lzPlane.getElements("hosts");
+                    ForEach(*hostsIter)
+                    {
+                        SocketEndpoint planeEp(hostsIter->query().queryProp(nullptr));
+                        if (planeEp.ipequals(destEp))
```

Comment:
Questions.  Should this be using a string compare (not sure)  and I think it should complain if it is not unique

Reply:
Hestitated when resolving 1st to IP before comparing, but figured it may be possible for there to be multiple DNS names (e.g. with and without domain) that would resolve to the same physical IP.

> I think it should complain if it is not unique

It is assumed by this code that is true. At least I am assuming it is invalid to define multiple 'lz' planes with the same or overlapping hosts (?).
Feels like that checking should live elsewhere though, not here?